### PR TITLE
Add to indesign button

### PR DESCRIPTION
**Pull request by rcrphillips**
Adds the ability to push a Composer article to InDesign instead of InCopy. The 'seen in incopy' stuff is still useful and relevant even if it's been pushed to indesign, which is why I have kept that contract there. 
screen shot 2017-10-23 at 17 40 36

#### Editorial tools integration tests
The editorial tools integration tests live [here](https://circleci.com/gh/guardian/editorial-tools-integration-tests) and get run every time workflow-frontend is deployed to the CODE environment. You should run them before merging this change to master. The current status of the tests (based off the last thing which was deployed to CODE) is:
[CircleCI](https://circleci.com/gh/guardian/editorial-tools-integration-tests)

### CODE REVIEW

File: public/components/content-list-drawer/content-list-drawer.js [hunk 1]
```diff
@@ -28,6 +28,13 @@ export function wfContentListDrawer($rootScope, config, $timeout, $window, conte
             });
     }
 
+    function buildIndesignUrl(fields) {
+        return config.indesignExportUrl
```
Comment (philmcmahon) [inline public/components/content-list-drawer/content-list-drawer.js]: You could avoid duplicating the logic in the indesign/incopy functions with a something like 
```
   function buildExportUrl(baseExportUrl) {
        return baseExportUrl
            .replace(/\$\{(.*?)\}/g, function(match, fieldName) {
                return fields[fieldName] || "";
            });
}
```

File: public/app.js [hunk 2]
```diff
@@ -76,6 +76,11 @@ angular.module('workflow',
                    "|^\\s*" + _wfConfig.incopyExportUrl.match("^.*?:")[0])
         );
 
+        $compileProvider.aHrefSanitizationWhitelist(
```
Comment (philmcmahon) [inline public/app.js]: same as below - you could avoid duplicated logic here by having a function which takes the export url as a parameter

**Comment by philmcmahon** [comment]
Looks good! Just a couple of things to fix.

File: public/app.js (hunk 2 above)
Comment (rcrphillips) [reply public/app.js]: @philmcmahon Ok, I gave it a shot myself, but need your help with these.

Comment (philmcmahon) [reply public/app.js]: Sorry, I will help out tomorrow!

**Comment by rcrphillips** [comment]
@philmcmahon could you double check this again?

**Comment by philmcmahon** [comment]
👍

**Pull request merged**